### ISSUE TITLE
It is not possible to use 'or' and 'and' when comparing attribute of a relation 

### ISSUE THREAD

**Issue by qria** [jfinkels/flask-restless#435]
I'm not very familiar with SQL and SQLalchemy so I might be confused about something or might be asking something impossible.

Say, table 'content' and 'date' has one to many relationship and I want to load all contents which contains date(time) in a specific date(time) range. But, 'any' operator does not support multiple options, and using 'and' outside of 'any' operator does not work since it will mean a different query. 

One possible syntax might be :  
```
http://127.0.0.1:5000/api/content?q={"filters":[
	{"name":"date_list","op":"any","val":
		{"and":[
			{"name":"date","op":">=","val":"2015-05-10"},
			{"name":"date","op":"<","val":"2015-05-11"}
			]
		}
	}
]}
```

On a side note, does anyone know an alternative implementation of how I can achieve this?

**Comment by samtux** [comment]
I am also looking for a solution and I could not find it. Please someone could help if such filters is that possible?

**Comment by jfinkels** [comment]
I have updated the code, please check if this is still a problem in the development version. See http://flask-restless.readthedocs.org/en/latest/fetching.html#filtering for the documentation.

**Comment by jfinkels** [comment]
This is still an issue, I placed a failing unit test in pull request #487.

**Comment by jfinkels** [comment]
This is fixed due to pull request #532. I will add the unit tests from #487 shortly.
